### ISSUE TITLE
ESP32 Wont read mqtt messages after some time.

### ISSUE THREAD

**Issue by gannaramu** [collin80/AWS_IOT#2]
is there any logic to run in the void loop() to keep the mqtt client connected?

**Comment by collin80** [comment]
No, there should not be any required function to call in loop. It's been a while since I tried to use the library. I remember we had a bit of trouble with stability and ended up moving away from directly using amazon IoT on the ESP32. But, I don't remember what the exact problem was. Perhaps it was disconnects.

**Comment by gannaramu** [comment]
That's True. It disconnects and I tried to have some periodic function that would reconnect but that didn't work too.
